woocommerce/woocommerce-android
# [Order Creation] Customer list - show cached list if available

Closes: #9828


### Description

The PR brings caching of the first page of the customers for the WCCore 8+. That shows cached customers right away, while fetching the latest date in the background.

Comes with Fluxc PR - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2859

### Testing instructions

Test this on sites with WCCore 8+ and WCCore versions lower than 8

* Orders -> Create Order -> Add customer
* If that's done for the first time, notice that the list is loading
* For the second and further times, notice that the list is shown right away

### Images/gif



https://github.com/woocommerce/woocommerce-android/assets/4923871/1fd16472-0044-48c1-a7ba-5af8a630af5f
